README.md

```diff
@@ -62,20 +62,23 @@ On the Raspberry Pi:
 1. Set up environment
    * Update package sources: `sudo apt update`
    * Make sure git is installed: `sudo apt install git`
-   * Make sure pip is installed: `sudo apt install python3-pip`
-2. Install Waveshare e-paper drivers
+   * Make sure pip is installed: `sudo apt install python3-pip python3-venv`
+2. Create & activate python virtual environment (venv)
+   * Create virtual environment: `python3 -m venv .venv`
```

Comment:
when creating virtual environment need the `--system-site-packages` directive like in the install script

Reply:
You're correct, it's in the actual function I added to the installer was just a miss on the README.md changes to do it manually.